src/pretix/control/views/item.py

```diff
@@ -283,6 +283,22 @@ def get_queryset(self):
         return self.request.event.questions.prefetch_related('items')
 
 
+def reorder_questions(request, organizer, event):
```

Comment:
There's a ``@event_permission_required("can_change_items")`` missing here. Tip: Also add the URL to ``tests/control/test_permissions.py``